### PR TITLE
Us10 kara

### PR DESCRIPTION
**Pull request by Karshyga1985**

### CODE REVIEW

**Review by celiker57 — APPROVED** [review]
Looks all good. Good work Kara.

File: src/test/resources/features/login.feature [hunk 1]
```diff
@@ -6,8 +6,8 @@ Feature: Users should be able to login
 
 @wip
   Scenario: Verify login with different user types
-    Given the user logged in as "<userType>"
- #   Given the user logged in with username as "User1" and password as "UserUser123"
+    #Given the user logged in as "<userType>"
```
Comment (Mykola2024) [inline src/test/resources/features/login.feature:9]: Why are you commenting out this step?
I am using this in my feature file

File: src/test/resources/features/US_10.feature
Comment (Mykola2024) [inline src/test/resources/features/US_10.feature:1]: Can you rename your Feature file and stepDefs file to follow our naming convetion
Screenshot 2024-03-19 at 3 20 40 PM

**Review by Mykola2024 — CHANGES_REQUESTED** [review]
Please address comments.

**Review by Mykola2024 — APPROVED** [review]

**Pull request merged**